tests/widgets_block.rs

```diff
@@ -208,182 +205,121 @@ fn widgets_block_title_alignment() {
             terminal
                 .draw(|frame| frame.render_widget(block, area))
                 .unwrap();
-            terminal.backend().assert_buffer(&expected);
+            terminal.backend().buffer().assert_eq(expected);
         }
     }
 
     // title top-left with all borders
     test_case(
         Alignment::Left,
         Borders::ALL,
-        Buffer::with_lines(vec![
-            " ┌Title──────┐ ",
-            " │           │ ",
-            " └───────────┘ ",
-        ]),
+        &Buffer::with_lines([" ┌Title──────┐ ", " │           │ ", " └───────────┘ "]),
```

Comment:
format break (I'm not going to mark more of these, perhaps run a regex over the diff to find them all.